packages/node/src/indexer/indexer.manager.ts

```diff
@@ -174,6 +174,8 @@ export class IndexerManager {
   }
 
   private async ensureMetadata(schema: string) {
+    type KeyValue = { [key: string]: string | number | boolean };
```

Comment:
type definition shouldn't be inside a function. Move it to top level (out of class)